### PR TITLE
Issue #4 Add RTK to project

### LINKED ISSUE #4: Add redux and redux-toolkit for managing data
Installation guide can be found here: https://redux-toolkit.js.org/tutorials/quick-start
Wrap App component with Provider
Add empty store

### PR DESCRIPTION
**Pull request by dobbi-free**

### CODE REVIEW

File: src/store/store.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { configureStore } from "@reduxjs/toolkit";
+
+export const store = configureStore({
+  reducer: {},
+  devTools: true,
```
Comment (wight554) [inline src/store/store.ts]: Need to make it true only for dev environment, it's usually process.env.NODE_ENV === 'development' , but I'd double check how it's handled in vite

Comment (wight554) [reply src/store/store.ts]: Upd.
`import.meta.env.DEV: {boolean} whether the app is running in development (always the opposite of import.meta.env.PROD) `

Comment (dobbi-free) [reply src/store/store.ts]: done

**Review by wight554 — APPROVED** [review]
lgtm

**Pull request merged**